area_detector_handlers/handlers.py

```diff
@@ -317,3 +318,130 @@ def get_file_list(self, datum_kwargs_gen):
                 fn = self._template % (self._path, self._filename, j)
                 file_list.append(fn)
         return file_list
+
+
+def read_header(fp):
```

Comment:
`fp` strongly suggests "filepath" to me, but in fact this is expected to be a file. Can we make the parameter name `file?
```suggestion
def read_header(file):
```